spring-boot-project/spring-boot-starters/README.adoc

```diff
@@ -241,4 +241,9 @@ do as they were designed before this was clarified.
 | https://github.com/knowm/XChange[XChange]
 | https://github.com/cassandre-tech/cassandre-trading-bot
 
+| https://groupe-sii.github.io/ogham/[Ogham]
+a| * https://github.com/groupe-sii/ogham/tree/master/ogham-spring-boot-starter-all
```

Comment:
For consistency with the other entries, could you please remove the bullet points?